.github/workflows/e2e_test.yml

```diff
@@ -60,7 +61,25 @@ jobs:
             dataset_config_name=wikitext-2-raw-v1 \
             profile_step=3 \
             max_steps=15
-
+      - name: Run Llama 3.1 8B (Splash Attention)
+        id: run-llama-3_1-8b-SplashAttention
+        env:
+          HF_TOKEN: ${{ secrets.HF_TOKEN }}
+          XLA_IR_DEBUG: 1
+          XLA_HLO_DEBUG: 1
+        run: |
+          name=$(e2e_testing/gen_name.py llama-3.1-8b)
```

Comment:
nit: maybe `name=$(e2e_testing/gen_name.py llama-3.1-8b-sa)` so that the SA and FA run artifacts are separated in the GCS buckets